administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -51,4 +49,5 @@ CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
   KEY `idx_tour` (`tour_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
 
-
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
+(0, 'com_guidedtours', 'component', 'com_guidedtours', '', 1, 1, 0, 1, '', '{}', '', 0, NULL, 0, 0);
```

Comment:
```suggestion
(0, 'com_guidedtours', 'component', 'com_guidedtours', '', 1, 1, 0, 1, '', '{}', '', 0, NULL, 0, 0);

```

Reply:
It's better to have a new line at the end of the file, therefore the above suggestion.

Beside this, it needs to insert the record for that new component into the `#__extensions` table for new installations. This has to be done by appending it to the insert statement here after this line https://github.com/joomla-projects/soc21_guided-tour/blob/dev/installation/sql/mysql/base.sql#L182 so together with that line it looks like:

```
(0, 'com_mails', 'component', 'com_mails', '', 1, 1, 1, 1, 1, '', '', ''),
(0, 'com_guidedtours', 'component', 'com_guidedtours', '', 1, 1, 0, 1, 1, '', '{}', '');
```

Note that there are different columns ised in that statement compared to the update SQL script.

Reply:
Finally, I think that the extension should not be protected, which is is in your proposed SQL here. It should be locked so it cannot be uninstalled, which it is in my proposed change here, but it should not be protected so it still can be disabled.

My above suggestion and the comment here does not mention that, but you should be able to change that on your own.